pkg/api/openshiftcluster.go

```diff
@@ -192,12 +192,16 @@ type ServicePrincipalProfile struct {
 	SPObjectID   string       `json:"spObjectId,omitempty"`
 }
 
+// SDNProvider
+type SDNProvider string
```

Comment:
We need to define constants for possible values, I think. Similar to external representations.

Reply:
https://github.com/Azure/ARO-RP/pull/1571/commits/7cf26fefa84635caced03ddb15651f0635027f4f#diff-f9a5398ee035bbae6c18fc8468081bbbd9bab4d81504809a5589541ca2171aa7R198-R202

Defined